plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/messages.properties

```diff
@@ -1,12 +1,14 @@
+BUTTON_ACCOUNTS_PANEL_ADD_ACCOUNT=Add Account
+BUTTON_ACCOUNTS_PANEL_LOGOUT=Sign Out
 LOGIN_ERROR_CANNOT_OPEN_BROWSER=Error opening a system browser. Signing in requires an open browser.
 LOGIN_ERROR_DIALOG_MESSAGE=An error occurred while trying to sign in.
 LOGIN_ERROR_DIALOG_TITLE=Sign in Failed
 LOGIN_ERROR_LOCAL_SERVER_RUN=Error listening for authorization code:\n\n
-LOGIN_MENU_LOGGED_IN=Sign out from Google
+LOGIN_MENU_LOGGED_IN=Sign in to/Sign out from Google...
```

Comment:
Okay, I tried to implement dynamically switching between one menu item ("sign in...") and two menu items ("add account..." and "sign out...") depending on the login status, but it got a bit ugly. @elharo Do you think having two menu items is a way to go?